diff --git a/os_dep/osdep_service.c b/os_dep/osdep_service.c
index 3b927f4..388fb8d 100644
--- a/os_dep/osdep_service.c
+++ b/os_dep/osdep_service.c
@@ -20,6 +20,10 @@
 
 #define RT_TAG '1178'
 
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 4, 0)
+MODULE_IMPORT_NS(VFS_internal_I_am_really_a_filesystem_and_am_NOT_a_driver);
+#endif
+
 #ifdef DBG_MEMORY_LEAK
 #ifdef PLATFORM_LINUX
 atomic_t _malloc_cnt = ATOMIC_INIT(0);
